pkg/modules/observability.go

```diff
@@ -291,6 +291,7 @@ func getTopology(op utils.OperatorConfig, cr csmv1.ContainerStorageModule) (stri
 		}
 	}
 
+	YamlString = strings.ReplaceAll(YamlString, CSMName, cr.Name)
```

Comment:
@abhi16394 I saw you declared a variable "CSMName = "<NAME>"" in application_mobility.go . I don't see the same declaration in Observability.go and commonconfig.go. Will it append the attribute to yaml if it does not exist ?

Reply:
since the variable declaration is global and all other files come under package module, 
We dont have to redeclare the variable "CSMName" again in other files.